app/src/lib/shells/linux.ts

```diff
@@ -1,26 +1,81 @@
 import { spawn } from 'child_process'
+import * as fs from 'fs'
+import { assertNever, fatalError } from '../fatal-error'
 import { IFoundShell } from './found-shell'
 
 export enum Shell {
   Gnome = 'gnome-terminal',
+  Tilix = 'tilix',
```

Comment:
Should this be the proper name `Tilix` here?

Reply:
I've used `tilix` just because you were already using `gnome-terminal`, which is the cli command to spawn a new gnome-terminal. 
If we use `Tilix`, shouldn't we also use `Terminal` and not `gnome-terminal`?

Reply:
This only really matters when we display the menu items, and here's what it looks like for macOS:

<img width="466" src="https://user-images.githubusercontent.com/359239/31861396-d95a5f16-b76f-11e7-8399-def6a4f2b8ac.png">

I think it was a mistake to name this `gnome-terminal`, and it could easily be named `GNOME Terminal` or `Terminal` here. I'll leave it to you to figure out which you prefer.